### PR TITLE
Module compilation timing visualization

### PR DESCRIPTION
**Pull request by wavewave**
The module compilation timing is visualized as online SVG image.

### CODE REVIEW

File: app/toolbox-server/Toolbox/Render.hs [hunk 1]
```diff
@@ -70,16 +71,17 @@ renderMainPanel ::
...
-    CheckImports -> renderInbox (CheckImports, mexpandedModu) m
+    CheckImports -> renderInbox (CheckImports, mexpandedModu) (serverInbox ss)
     Timing ->
       div
         []
-        [ pre [] [text $ T.pack $ show s]
-        , renderInbox (Timing, mexpandedModu) m
+        [ pre [] [text $ T.pack $ show (serverSessionInfo ss)]
+        , div [] [renderTiming ss]
+        -- , renderTiming ss -- renderInbox (Timing, mexpandedModu) (serverInbox ss)
```
Comment (halogenandtoast) [inline app/toolbox-server/Toolbox/Render.hs:82]: Is this leftover?

**Review by halogenandtoast — APPROVED** [review]
Overall looks good, seems like one leftover comment. For my understanding what does `SN` represent?

File: app/toolbox-server/Toolbox/Render.hs (hunk 1 above)
Comment (wavewave) [reply app/toolbox-server/Toolbox/Render.hs:82]: oops. missed this remained in this PR. the following PR #13 removed this line. Thanks for the catch.

**Comment by wavewave** [comment]
Thanks for the review. SN  is message serial number, by which the message inbox checks whether a new message came.

**Pull request merged**